app/views/login.html

```diff
@@ -0,0 +1,179 @@
+
+
+                            {% if loginError %}
+                            <div class="alert alert-dismissable alert-danger">
+                                <button type="button" class="close" data-dismiss="alert">&times;</button>
+                                {{loginError}}
+                            </div>
+                            {% endif %}
+
+
+
+                            <form method="post" role="form" method="post" id="loginform">
+                                <div class="form-group">
+                                    <label for="userName">User Name</label>
+                                    <input type="text" class="form-control" id="userName" name="userName" value="{{userName}}" placeholder="Enter User Name">
+                                </div>
+
+                                <div class="form-group">
+                                    <label for="password">Password</label>
+                                    <input type="password" class="form-control" id="password" name="password" value="{{password}}" placeholder="Enter Password">
+                                </div>
+                                <input type="hidden" name="_csrf" value="{{csrftoken}}" />
+
+                                <div class="row">
+                                    <div class="col-lg-4"><a href="/signup">New user? Sign Up</a>
+                                    </div>
+                                    <div class="col-lg-5"></div>
+                                    <div class="col-lg-3">
+                                        <button type="submit" class="btn btn-danger">Submit</button>
+                                    </div>
+                                </div>
+                            </form>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-lg-4"></div>
+            </div>
+            <!-- /.row -->
+
+        </div>
+        <!-- /#page-wrapper -->
+        <!-- /.row -->
+    </div>
+    <!-- /#wrapper -->
+
+    <!-- Bootstrap core JavaScript -->
+    <script src="/vendor/jquery.min.js"></script>
+    <script src="/vendor/bootstrap/bootstrap.js"></script>
+    <!-- Load environmental scripts such as live reload -->
+    {% for script in environmentalScripts %}
+    {{script}}
+    {% endfor %}
+    <script type="application/javascript">
+    const areCookiesEnabled = () => {
+        const cookieEnabled = navigator.cookieEnabled;
+
+        // When cookieEnabled flag is present and false then cookies are disabled.
+        if (!cookieEnabled) return false;
+
+        // try to set a test cookie if we can't see any cookies and we're using
+        // either a browser that doesn't support navigator.cookieEnabled
+        // or IE (which always returns true for navigator.cookieEnabled)
+        if (!document.cookie && cookieEnabled === null) {
+            document.cookie = "testcookie=1";
```

Comment:
#### Automated PR Comment From Polaris SAST
<!-- ef8438255f1698ce17829d41ef57021d -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Polaris SAST Issue - Unreachable, Unused or Dead Code
_Medium CWE-561_
Code can never be reached because of a logical contradiction

The indicated dead code may have performed some action; that action will never occur.

### How to fix
Unreachable and dead code should be analyzed and either removed or corrected depending on whether it is needed. If the code should execute but doesn't, any bugs preventing the code from executing must be fixed. If the code is not needed, and its removal will not have a negative impact on the application, it should be removed.